src/MSAL.PCL/TokenCacheNotificationArgs.cs

```diff
@@ -28,33 +28,43 @@
 namespace Microsoft.Identity.Client
 {
     /// <summary>
-    /// Contains parameters used by the ADAL call accessing the cache.
+    /// 
     /// </summary>
-    public sealed class TokenCacheNotificationArgs
+    public enum NotificationType
     {
         /// <summary>
-        /// Gets the TokenCache
+        /// 
         /// </summary>
-        public TokenCache TokenCache { get; internal set; }
+        RefreshTokenCache,
+        /// <summary>
+        /// 
+        /// </summary>
+        TokenCache
```

Comment:
This should not even be here. I will delete until we find a better way to do this

---
In reply to: [93710625](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/138#discussion_r93710625) [](ancestors = 93710625)